tests/test_utils/test_atoms.py

```diff
@@ -0,0 +1,74 @@
+import lalpulsar
+import pytest
+
+from pyfstat.utils.atoms import (
+    copy_FstatAtomVector,
+    extract_singleIFOmultiFatoms_from_multiAtoms,
+)
+
+
+@pytest.fixture
+def arbitrary_singleAtoms():
+    single_atoms = lalpulsar.CreateFstatAtomVector(5)
+    for i in range(single_atoms.length):
+
+        for attr in ["TAtom"]:
+            setattr(single_atoms, attr, i)
```

Comment:
would work, but it's confusing as not matching standard usage:
```suggestion
        single_atoms.TAtom = 1800
```

Reply:
...better out-of-loop